src/frontend/ui/ui_manager.rs

```diff
@@ -49,17 +53,25 @@ pub struct UiManager {
 
     #[derive_where(skip(Debug))]
     render_observer: Vec<Box<RenderObserver>>,
+
+    config: Config,
 }
 
 impl UiManager {
-    pub fn new(total_width: i32, total_height: i32, app_model_sender: Sender<Command>) -> Self {
+    pub fn new<P: AsRef<Path>>(
+        total_width: i32,
+        total_height: i32,
+        app_model_sender: Sender<Command>,
+        cfg_path: P,
+    ) -> Self {
         UiManager {
             tool_manager: ToolManager::new(),
             canvas: Canvas::new(total_width, total_height).expect("Couldn't create canvas."),
             selection: Rectangle::with_size(total_width as f64, total_height as f64),
             drawables: vec![],
             render_observer: vec![],
             app_model_sender,
+            config: Config::load(cfg_path).unwrap_or_default(),
```

Comment:
maybe pass the config as argument instead of letting the ui manager parse => do not bind on loading a file.